src/azul/plugins/metadata/hca/service/response.py

```diff
@@ -365,6 +365,9 @@ def make_projects(self, entry) -> MutableJSONs:
                 'projectShortname': project['project_short_name'],
                 'laboratory': sorted(set(project.get('laboratory', [None]))),
                 'estimatedCellCount': project['estimated_cell_count'],
+                'isTissueAtlasProject': project.get('is_tissue_atlas_project'),
+                'tissueAtlas': project.get('tissue_atlas'),
+                'bionetworkName': project.get('bionetwork_name')
```

Comment:
What warrants the use of `.get`. Under what circumstances can any of these properties be missing?

Reply:
I don't think there are any.

Reply:
Correction: `get` is not needed for `isTissueAtlasProject` or `bionetworkName` but it is needed for `tissueAtlas` because that field is not populated for outer entity types other than `projects`.